src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -28,13 +34,23 @@ public ToastNotifier()
 
 		private string ConvertToastTextToString(string toasttext)
 		{
+			var uri = Android.Net.Uri.Parse(toasttext);

```

Comment:
Please `System.Uri.TryParse`, not the native android uri parser.

Reply:
According to doc, https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=net-6.0 , there is no System.Uri.TryParse method. It can be only by constructor Uri(string), or Uri.TryCreate(string, 0, out uri)

Reply:
Yes, indeed, it was `Uri.TryCreate`, my bad. Also, don't resolve the conversation if you find the comment to be unclear (even if it was wrong in this case).

Reply:
I reverted to previous (original) implementation.
Using Uri (in any sense) is bad, as 'toasttext' can be e.g. "no", "", "<show>", "do not use this" etc., and when it starts with prefix, it cannot be parsed as uri. E.g. "ms-resource:text".

Reply:
Can this conversation be marked as resolved?

Reply:
Can this conversation be marked as resolved?

Reply:
Yes